### ISSUE TITLE
Extension - Dev tools

### ISSUE THREAD

**Issue by Tamayuzu** [MeD-DMC/Adapt-accessibilityfixes#180]
Button doesn't have a name/aria-label

**Comment by ftourangeau** [comment]
Duplicate of #179
